jxl/src/features/spline.rs

```diff
@@ -439,7 +445,132 @@ fn for_each_equally_spaced_point<F: FnMut(Point, f32)>(
     f(points[points.len() - 1], accumulated_distance);
 }
 
-const DESIRED_RENDERING_DISTANCE: f32 = 1.0;
+trait Dct32 {
+    fn continuous_idct(&self, t: f32) -> f32;
+}
+
+impl Dct32 for [f32; 32] {
+    fn continuous_idct(&self, t: f32) -> f32 {
```

Comment:
Why is this a trait instead of a free function?

Reply:
I just like the aesthetic, but I can change it to a function. WDYT?

Reply:
I prefer functions unless there's a good reason to go with traits :-)

Reply:
Done